.github/workflows/gradle-build.yml

```diff
@@ -22,4 +22,7 @@ jobs:
         uses: gradle/gradle-build-action@v2
 
       - name: Run Tests
+        env:
+          MAVEN_USERNAME: user
+          MAVEN_PASSWORD: password
         run: ./gradlew test
```

Comment:
The `MAVEN_USER` and `MAVEN_PASSWORD` are set to dummy values so gradle can validate the values successfully without error.